src/commands/services/list.ts

```diff
@@ -246,6 +253,16 @@ export default class ServicesList extends Command {
           isVisible: {
             header: "is_visible"
           },
+          isQuality: {
+            header: "is_quality",
+            get: row =>
+              SERVICEID_EXCLUSION_LIST.indexOf(row.serviceId) === 0
+                ? true
+                : ValidService.decode(row).fold(
+                    _ => false, // quality ok
+                    _ => true // quality ko
```

Comment:
```suggestion
                : ValidService.decode(row).isRight()
```
Could it be more coincise?